Explorer/src/components/sidebar.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
+
+import classNames from "classnames";
+import { Sidebar as FlowbiteSidebar } from "flowbite-react";
+import type { FC, PropsWithChildren } from "react";
+import { useSidebarContext } from "@/context/sidebar_context";
+import {
+  HiPresentationChartLine,
+  HiCog,
+  HiFolder,
+  HiServer,
+  HiTemplate,
+  HiChip,
+  HiTable,
+  HiHome,
+} from "react-icons/hi";
+
+const Sidebar: FC<PropsWithChildren<Record<string, unknown>>> = function ({
+  children,
+}) {
+  const { isOpenOnSmallScreens: isSidebarOpenOnSmallScreens } =
+    useSidebarContext();
+
+  return (
+    <div
+      className={classNames(
+        "fixed overflow-auto top-0 h-screen z-10 lg:sticky lg:!block",
+        {
+          hidden: !isSidebarOpenOnSmallScreens,
+        }
+      )}
+    >
+      <FlowbiteSidebar>{children}</FlowbiteSidebar>
+    </div>
+  );
+};
+
+/*import { useNavigationStore } from "@/context/sidebar_context";*/
+
+export default Object.assign(Sidebar, { ...FlowbiteSidebar });
+
+export function ActualSidebar(): JSX.Element {
+  const current_page = "test"; /*useNavigationStore((state) => state.page);*/
+
+  return (
+    <FlowbiteSidebar>
+      <FlowbiteSidebar.Items>
+        <FlowbiteSidebar.ItemGroup>
+          <FlowbiteSidebar.Item
+            href="dashboard"
+            icon={HiPresentationChartLine}
+            active={current_page.localeCompare("dashboard") ? false : true}
```

Comment:
What about using [String.startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) 
```suggestion
            active={current_page.startsWith("/dashboard") ? false : true}
```
This way we also cover stuff like `/cluster/1` or smt like that.

Reply:
Also why the explicit ternary? If you want it to be inactive if the localCompare is true, then you can use `!current_page.localeCompare("dashboard")`, right?

Reply:
Accounted for this with the new change.